Manual.md: document requirement of tagged release in place linked from PR template (#29354)
void-linux/void-packages: Manual.md

Browser forks, including those based on Chromium and Firefox, are generally not
accepted. Such forks require heavy patching, maintenance and hours of build time.

> Package need to be provided in version announced by authors as ready to use by
> general public. Betas, arbitrary VCS revisions, templates using tip of development
> branch taken at build time and releases done by package maintainer won't be accepted.

Reviewer: ```suggestion
Packages should use versions announced by authors as ready to use by the
general public - usually called releases. Betas, arbitrary VCS revisions, templates using tip of development
branch taken at build time and releases done by package maintainer won't be accepted.
```

I would rather list "Alphas" than "Betas", because some upstreams tag a stable release every decade, but release "betas" in pretty good cadence, which are what everyone uses. We do have some exceptions in place, but idk how to document that.

Author: > rather list "Alphas" than "Betas"

I conciously made muddy what we accept, because of usage of pre-releases (wine) or arbitrary VCS revisions (nawk, iana-etc).

> should

I also prefer stronger wording here and handling some things when needed as _exceptions_ - rarely, when needed.

Reviewer 2: > I conciously made muddy what we accept, because of usage of pre-releases (wine) or arbitrary VCS revisions (nawk, iana-etc).

Also see mtpaint. The author does not tag releases - announces them in a commit message.

Reviewer: @Chocimier updated the suggestion, then.

Author: Thanks, updated.
